source/agora/node/admin/AdminInterface.d

```diff
@@ -319,13 +326,42 @@ public struct EncryptionKey
     public Hash value;
 }
 
+unittest
+{
+    import agora.utils.Test;
+    import agora.consensus.data.Params;
+    import agora.common.ManagedDatabase;
+
+    auto key_pair = WK.Keys.A;
+    auto clock = new Clock(null, null);
+    // create an EnrollmentManager object
+    auto params = new immutable(ConsensusParams)();
+    auto stateDB = new ManagedDatabase(":memory:");
+    auto cacheDB = new ManagedDatabase(":memory:");
+    auto man = new EnrollmentManager(stateDB, cacheDB, key_pair, new immutable(ConsensusParams)(20));
+    auto adminInterface = new AdminInterface(key_pair, clock, man);
+    string contentType = "image/svg+xml";
+    string vary = "Accept-Encoding";
+    string enc_qr_svg = adminInterface.encryptionKeyQR("Votera", 100, contentType, vary);
+
+    string sample_enc_qr_svg =
+`<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 500 500">
```

Comment:
This is not really testing anything. I'd rather we remove this and have the test on DevOps side.

Reply:
Okay, I removed this test code.